src/main/io/gps.c

```diff
@@ -2609,6 +2613,15 @@ void GPS_calculateDistanceAndDirectionToHome(void)
     }
 }
 
+void GPS_distances(const gpsLocation_t *from, const gpsLocation_t *to, float *pNSDist, float *pEWDist) {
```

Comment:
Please use GPS_distance_cm_bearing instead

Reply:
Why? We don't need to compute bearing.

Reply:
```suggestion
void GPS_distances(const gpsLocation_t *from, const gpsLocation_t *to, vector2_t* distCm) {
```

But it would be better to return distance in SI units